package.json

```diff
@@ -12,10 +12,6 @@
   },
   "dependencies": {
     "@emailjs/browser": "^4.3.3",
-    "dotenv": "^16.4.5",
```

Comment:
why you remove these (express, node, modemailer)        ?

Reply:
@codervivek5 I didn't remove it, It might be got removed while installing the necessary dependencies you said to me i.e, sweet library

Reply:
Ohh kindly install these also using npm so that other contributors will not face any issue

Reply:
Sure, i'll do this right away

Reply:
Ok let me know  when this will fix

Reply:
Hey @codervivek5 I have just checked it but It's already there, only line numbers are changed

![image](https://github.com/user-attachments/assets/c67a3ee3-a6f6-49ae-bf25-1343e78f2c20)

Reply:
Ohh ok then fine 👍